Classes/FrontEnd/RegistrationForm.php

```diff
@@ -771,13 +772,24 @@ public function populateListPaymentMethods(): array
             return [];
         }
 
-        $rows = \Tx_Oelib_Db::selectMultiple(
-            'uid, title',
-            'tx_seminars_payment_methods, tx_seminars_seminars_payment_methods_mm',
-            'tx_seminars_payment_methods.uid = tx_seminars_seminars_payment_methods_mm.uid_foreign ' .
-            'AND tx_seminars_seminars_payment_methods_mm.uid_local=' . $this->getSeminar()->getTopicOrSelfUid() .
-            \Tx_Oelib_Db::enableFields('tx_seminars_payment_methods')
-        );
+        /** @var \TYPO3\CMS\Core\Database\Query\QueryBuilder $qb */
+        $qb = GeneralUtility::makeInstance(ConnectionPool::class)
+            ->getQueryBuilderForTable('tx_seminars_payment_methods');
+        $rows = $qb
+            ->select('uid', 'title')
+            ->from('tx_seminars_payment_methods, tx_seminars_seminars_payment_methods_mm')
+            ->where(
+                $qb->expr()->eq(
+                    'tx_seminars_payment_methods',
+                    'tx_seminars_seminars_payment_methods_mm.uid_foreign'
+                ),
+                $qb->expr()->eq(
+                    'tx_seminars_seminars_payment_methods_mm.uid_local',
+                    $qb->createNamedParameter($this->getSeminar()->getTopicOrSelfUid(), \PDO::PARAM_INT)
+                )
+            )
```

Comment:
Please use explicit JOINs instead:
https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/ApiOverview/Database/QueryBuilder/#join-innerjoin-rightjoin-and-leftjoin